apps/src/templates/MultipleSectionsAssigner.jsx

```diff
@@ -159,54 +160,70 @@ const MultipleSectionsAssigner = ({
   };
 
   return (
-    <BaseDialog isOpen={true} handleClose={onClose}>
-      <div style={styles.header} className="uitest-confirm-assignment-dialog">
-        {i18n.chooseSectionsPrompt({assignmentName})}
-      </div>
-      <div style={styles.content}>{i18n.chooseSectionsDirections()}</div>
-      <div style={styles.header} className="uitest-confirm-assignment-dialog">
-        {i18n.yourSectionsList()}
-      </div>
-      <div style={styles.grid}>
-        {sections &&
-          sections.map(
-            section =>
-              isAssignableToSection(section.participantType) && (
-                <TeacherSectionOption
-                  key={section.id}
-                  section={section}
-                  isChecked={
-                    !!currentSectionsAssigned.some(s => s.code === section.code)
-                  }
-                  assignedSections={currentSectionsAssigned}
-                  onChange={() => handleChangedCheckbox(section)} // this function should update the state of multiple section assigner
-                  editedValue={section.isAssigned}
-                />
-              )
-          )}
-      </div>
-      <hr />
-      <a
-        style={styles.selectAllSectionsLabel}
-        onClick={selectAllHandler}
-        className="select-all-sections"
-      >
-        Select All
-      </a>
-      <div style={styles.buttonContainer}>
-        <Button
-          text={i18n.dialogCancel()}
-          onClick={onClose}
-          color={Button.ButtonColor.gray}
-        />
-        <Button
-          id="confirm-assign"
-          text={i18n.confirmAssignment()}
-          onClick={reassignSections}
-          color={Button.ButtonColor.orange}
-        />
-      </div>
-    </BaseDialog>
+    <>
+      <div style={styles.modalBackdrop} />
```

Comment:
Looks like we might wanna reuse this dialog in the future. Why not create a reusable component and use it here? (just like we have baseDialog currently)

Reply:
Yes, agreed! That will be the next PR –– extracting this out into its own component using SCSS module styling. I did this to keep the PR short for a proof of concept.